packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.stories.ts

```diff
@@ -0,0 +1,135 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnEnumTextTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { tableColumnTextTag } from '../../text';
+import { mappingTextTag } from '../../../mapping/text';
+import { sharedMappingValidityDescription } from '../../enum-base/tests/shared';
+
+const simpleData = [
+    {
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        status: 'fail',
+        isChild: true
+    },
+    {
+        firstName: 'Marge',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Bart',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: true
+    },
+    {
+        firstName: 'Greg',
+        lastName: '',
+        status: 'foo',
+        isChild: true
+    },
+    {
+        firstName: 'Frank',
+        lastName: '',
+        status: 'bar',
+        isChild: false
+    }
```

Comment:
1. These two entries were to display the behavior for rows that don't match a mapping. This is also part of why grouping is turned on. There's a behavior (being questioned by Molly in another thread) where the unmapped key value is shown in the group header instead of a blank. If we consider unmapped values more of a corner case, then I don't need to demonstrate it in the story.

2. I used to have a second column that used `isChild`, but I had removed it. Forgot to remove the record properties.

Reply:
Ah, of course, thanks,. Personally I'd vote that the unmapped case is uncommon enough that it isn't worth documenting here.